### DISCUSSION TITLE
Need ability to open second level popup (e.g. to open a child popup from the parent popup)

### DISCUSSION THREAD

**Discussion by Orgbrat** [CommunityToolkit/Maui#335] [New Feature Discussions]
It would be really helpful if you could open a Child Popup from a Parent Popup that has already been displayed so you could pass data back to the Parent from the Child. This does not seem to work with Windows and iOS (have not tried MacOS and Android) as the Child Popup will not display until you have closed the Parent Popup. Makes it a real pain to have to close one and then open another then keep going thru the same process.

Does anyone else think this is a needed feature?

**Comment by pictos** [comment]
@ahoefling do you know if that is possible?

**Reply by SkyeHoefling** [reply]
Does the MAUI implementation allow multiple popups or is it always just 1 single popup at any given time?

IIRC the XCT implementation supported multiple popups at once. If that is the case in the MAUI Toolkit then the downstream consumer would have to write code to manage state and passing data between the popups. I would argue it would go out of scope of the MAUI Toolkit.

**Reply by pictos** [reply]
@ahoefling 

> Does the MAUI implementation allow multiple popups or is it always just 1 single popup at any given time?

Right now looks like the implementation just allows 1 single popup per time.

> IRC the XCT implementation supported multiple popups at once. If that is the case in the MAUI Toolkit then the downstream consumer would have to write code to manage state and passing data between the popups. I would argue it would go out of scope of the MAUI Toolkit.

Not sure if does, if you look [here](https://github.com/xamarin/XamarinCommunityToolkit/issues/1764), looks like this feature is missing on some platforms. So I asked you to make sure that all platforms support such feature

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
